config/prod/TESLA-download-bucket.yaml

```diff
@@ -44,6 +44,6 @@ parameters:
 # For CI system (do not change)
 hooks:
   before_launch:
-    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/master/s3-bucket-v2.j2 --create-dirs -o templates/remote/s3-bucket-v2.j2"
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/v0.2.5/s3-bucket-v2.j2 --create-dirs -o templates/remote/s3-bucket-v2.j2"
```

Comment:
there's already a `v0.2.7` so let's set this to `v0.2.8` after we pin to the current version by merging https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/385 and #462